README.md

```diff
@@ -3,39 +3,42 @@
 
 ## [Learning Bitmark](/pages/learning-bitmark/README.md)
 
-**[Bitmark Property System Introduction](/pages/learning-bitmark/problem-we-are-trying-to-solve.md)**
+**[Property System Introduction](/pages/learning-bitmark/problem-we-are-trying-to-solve.md)**
 
-**[Bitmark Blockchain Introduction](/pages/learning-bitmark/bitmark-blockchain.md)**
+**[Blockchain Introduction](/pages/learning-bitmark/bitmark-blockchain.md)**
 
 **[Quick Start](/pages/learning-bitmark/quick-start/README.md)**
-* [Running a Bitmark Node](/pages/learning-bitmark/quick-start/simple-solution-for-node-setup.md#run-a-bitmark-node)
-* [Creating a Bitmark Account](/pages/learning-bitmark/quick-start/working-with-bitmarks/creating-bitmark-account.md)
-* [Registering Bitmark Certificates](/pages/learning-bitmark/quick-start/working-with-bitmarks/issuing-bitmarks.md)
-* [Transferring Bitmark Certificates](/pages/learning-bitmark/quick-start/working-with-bitmarks/transferring-bitmarks.md)
-* [Using Bitmark Shares](/pages/learning-bitmark/quick-start/working-with-bitmarks/using-bitmark-shares.md)
+* [Running a Bitmark Node](/pages/learning-bitmark/quick-start/simple-solution-for-node-setup.md)
+* [Working with Bitmark](/pages/learning-bitmark/quick-start/working-with-bitmarks/working-with-bitmark.md)
+* [Using the Bitmark app](/pages/learning-bitmark/quick-start/working-with-bitmarks/using-bitmark-app.md)
+* [Using the SDK](/pages/learning-bitmark/quick-start/working-with-bitmarks/using-sdk.md)
+* [Using the CLI](/pages/learning-bitmark/quick-start/working-with-bitmarks/using-cli.md)
+* [CLI Payment](/pages/learning-bitmark/quick-start/working-with-bitmarks/payment-for-bitmark-cli.md)
```

Comment:
To standardize these headers, this should be something like "Paying with the CLI".

Reply:
And I'm wondering if it should be a subhead under "Using the CLI"? (Just indented?)

Reply:
I would like to keep "Using the CLI" format to be consistent with "Using the Bitmark app" & "Using the SDK". They only contain the instructions to execute bitmark transactions. "CLI Payment" is an additional document of setup instructions, it is referred to in both "Using the CLI" and "Using the Bitmark shares". I am thinking it might be under the "Bitmark references/ CLI" section. What do you think?

Reply:
Yes, that makes sense.